src/components/ui/CloseButton.jsx

```diff
@@ -0,0 +1,14 @@
+/* global _ */
+import React from 'react';
+import cx from 'classnames';
+
+const CloseButton = ({ className, onClick }) =>
```

Comment:
:+1: If we need to handle multiple buttons with only an icon later, I think we would add the relevant behavior into `Button.jsx`

Reply:
`<Button>` already supports the case of icon-only buttons (like on POI actions), but with a standard bordered style and no need to 'cheat' to expand the visible area. I think the case of close button is specific-enough to always be managed separately.

Reply:
as long as it is specific to close button yes